api/local.env

```diff
@@ -45,13 +45,22 @@ API_AUTH_TOKEN=LOCAL_AUTH_12345678,LOCAL_AUTH_87654321,LOCAL_1234
 
 LOGIN_GOV_CLIENT_ID=local_mock_client_id
 
-LOGIN_GOV_JWK_ENDPOINT=http://host.docker.internal:5001/issuer1/jwks
+LOGIN_GOV_JWK_ENDPOINT=http://mock-oauth2-server:5001/issuer1/jwks
 LOGIN_GOV_AUTH_ENDPOINT=http://localhost:5001/issuer1/authorize
-LOGIN_GOV_TOKEN_ENDPOINT=http://host.docker.internal:5001/issuer1/token
-LOGIN_GOV_ENDPOINT=http://host.docker.internal:5001/issuer1
+LOGIN_GOV_TOKEN_ENDPOINT=http://mock-oauth2-server:5001/issuer1/token
+LOGIN_GOV_ENDPOINT=http://mock-oauth2-server:5001/issuer1
 
+############################
+# OAuth Flow Debug Callback (Default)
+# Remove the comment under the Application Callback to redirect the user to simpler.grants.gov instead
+############################
 LOGIN_FINAL_DESTINATION=http://localhost:8080/v1/users/login/result
 
+############################
+# simpler.grants.gov Application Callback
+############################
+# LOGIN_FINAL_DESTINATION=http://localhost:3000/api/auth/callback
```

Comment:
Terminology here might be confusing/misleading. I wouldn't say "redirect to simpler.grants.gov", because it's not. I'd distinguish that more in a frontend/backend API manner.

I wouldn't put these under separate headings either (makes them seem like they're separate values entirely), just put a comment saying if you want to redirect to your local frontend you can override to the localhost:3000 path.